bukkit/src/main/java/me/leoko/advancedban/bukkit/listener/InternalListener.java

```diff
@@ -15,28 +15,44 @@
  * @author Beelzebu
  */
 public class InternalListener implements Listener {
-    
+
+    private void ban(BanList banlist, PunishmentEvent e) {
+        try {
+            banlist.addBan(e.getPunishment().getName(), e.getPunishment().getReason(), new Date(e.getPunishment().getEnd()), e.getPunishment().getOperator());
+        } catch (NullPointerException ex) {
+            Bukkit.getLogger().severe("No player is known by the name '" + e.getPunishment().getName() + "'");
+        }
+    }
```

Comment:
Can this be further specified to only catch when the error originates from:
> Cannot invoke "com.mojang.authlib.GameProfile.getId()" because "var0" is null
As this may catch-all issues that are not related to this issue. 

Thanks for your help!

Reply:
I can give you access to the PR if you want as I'm not able to fix this within the next week.

Reply:
I think I now fixed it better. The problem is that I don't want to check for a certain text in the error message. Neither would I like to go parsing the stack trace. Nor do something with reflection.

The negative thing with the current implementation is that a user has to have been logged on before he's allowed to be banned...

Neither option truly satisfies me, even more because there is no real internal method that I can query (without the use of reflection / fork-specific calls) to check the existence of the username.